packages/create-platformatic/src/ghaction.mjs

```diff
@@ -60,16 +107,66 @@ export const createGHAction = async (logger, env, type, projectDir, buildTS) =>
   }
 }
 
-/* c8 ignore next 12 */
-export const askCreateGHAction = async (logger, env, type, buildTS, projectDir = process.cwd()) => {
-  const { githubAction } = await inquirer.prompt([{
-    type: 'list',
-    name: 'githubAction',
-    message: 'Do you want to create the github action to deploy this application to Platformatic Cloud?',
-    default: true,
-    choices: [{ name: 'yes', value: true }, { name: 'no', value: false }]
-  }])
+/* c8 ignore next 19 */
+export const askDynamicWorkspaceCreateGHAction = async (logger, env, type, buildTS, projectDir = process.cwd()) => {
+  const { githubAction, workspaceId } = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'githubAction',
+      message: 'Do you want to create the github action to deploy this application to Platformatic Cloud dynamic workspace?',
+      default: true,
+      choices: [{ name: 'yes', value: true }, { name: 'no', value: false }]
+    },
+    {
```

Comment:
This should be a separate step, otherwise it always ask for workspaceId. It should also be possilble to trigger with a condition